### PR TITLE
LPS-185206 Remove FF endpoint to post Notification Queue Entry

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPS-185206](https://issues.liferay.com/browse/LPS-185206)

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

File: modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/resource/v1_0/NotificationQueueEntryResourceImpl.java [hunk 1]
```diff
@@ -225,11 +220,8 @@ private NotificationQueueEntry _toNotificationQueueEntry(
 					serviceBuilderNotificationQueueEntry.
 						getNotificationQueueEntryId();
 
-				if (FeatureFlagManagerUtil.isEnabled("LPS-178816")) {
-					recipients = notificationType.toRecipients(
-						notificationRecipient.
-							getNotificationRecipientSettings());
-				}
+				recipients = notificationType.toRecipients(
+					notificationRecipient.getNotificationRecipientSettings());
```
Comment (carolmariaabb) [inline modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/resource/v1_0/NotificationQueueEntryResourceImpl.java:225]: @pedro-oliveira446 please remove the empty lines

```
id =
	serviceBuilderNotificationQueueEntry.
		getNotificationQueueEntryId();
recipients = notificationType.toRecipients(
	notificationRecipient.
		getNotificationRecipientSettings());
recipientsSummary = notificationType.getRecipientSummary(
	serviceBuilderNotificationQueueEntry);
```

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by murilostodolni — APPROVED** [review]
LGTM 🎉

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by pedro-oliveira446** [comment]
ci:forward:force

**Pull request closed without merging**